rds-update-rdsh-collection/azuredeploy.json

```diff
@@ -0,0 +1,345 @@
+            "allowedValues": [
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Basic_A1",
+                "Basic_A2",
+                "Basic_A3",
+                "Basic_A4",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14",
+                "Standard_D1_v2",
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D4_v2",
+                "Standard_D5_v2",
+                "Standard_D11_v2",
+                "Standard_D12_v2",
+                "Standard_D13_v2",
+                "Standard_D14_v2",
+                "Standard_DS1",
+                "Standard_DS2",
+                "Standard_DS3",
+                "Standard_DS4",
+                "Standard_DS11",
+                "Standard_DS12",
+                "Standard_DS13",
+                "Standard_DS14",
+                "Standard_G1",
+                "Standard_G2",
+                "Standard_G3",
+                "Standard_G4",
+                "Standard_G5",
+                "Standard_GS1",
+                "Standard_GS2",
+                "Standard_GS3",
+                "Standard_GS4",
+                "Standard_GS5"
+            ]
+        },
+
+        "userLogoffTimeoutInMinutes": {
+            "type": "int",
+            "defaultValue": 60,
+            "metadata": {
+                "description": "Timeout before forced user logoff from old RDSH instances"
+            }
+        },
+
+        "adDomainName": {
+            "type": "string",
+            "metadata": {
+                "description": "The FQDN of the AD domain. For example contoso.com"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of administrator in the existing RDS deployment"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Administrator password."
+            }
+        },
+
+        "vnetName": {
+            "type": "string",
+            "defaultValue": "vnet",
+            "metadata": {
+                "description": "VNET name."
+            }
+        },
+        "subnetName": {
+            "type": "string",
+            "defaultValue": "Subnet",
+            "metadata": {
+                "description": "Subnet name."
+            }
+        }
+    },
+
+
+    "variables": {
+
+        "apiVersion": "2015-06-15",
```

Comment:
fixed this.
are you saying all templates using "apiVersion": "[variables('apiVersion')]",  doing this wrong?  
I've seen so many places where one apiVersion is used across whole template... but I would agree: each resource has it's own version list